Swift/Tests/MessageEndpointTest.swift

```diff
@@ -21,7 +21,7 @@ import XCTest
 import MultipeerConnectivity
 import CouchbaseLiteSwift
 
-protocol MultipeerConnectionDelegate: class {
+protocol MultipeerConnectionDelegate: AnyObject {
```

Comment:
What is the difference b/w AnyObject and class here? What is the issue with class?

Reply:
xcode deprecation warning - for class-constrained protocols, AnyObject should be used instead of class.